packages/@aws-cdk/aws-rds/README.md

```diff
@@ -43,6 +43,17 @@ By default, the master password will be generated and stored in AWS Secrets Mana
 Your cluster will be empty by default. To add a default database upon construction, specify the
 `defaultDatabaseName` attribute.
 
+### Enabling S3 integration to a cluster
+
+To enable S3 integration on a clustered database, set an IAM role with the required S3 policies to the s3ImportRole property. Remember to also set this role as the aws_default_s3_role, aurora_load_from_s3_role and/or aurora_select_into_s3_role in the cluster's parameter group.
```

Comment:
Use line width of 120.

Can you link to documentations where a user can learn more about what this s3 integration is and how to set it up? Usually, written as "Read more about S3 integration [here](link)".

Can you also link to details on why the - `aws_default_s3_role`, `aurora_load_from_s3_role` and `aurora_select_into_s3_role` need to be set? 
The latter two suggests that this is an Aurora only feature. Is that the case? Is this supported for all database types?

Reply:
It is indeed Aurora only. I've now added that to the readme. Also included links to the docs for both import and export.